test/e2e/util_gatewayapi_test.go

```diff
@@ -1037,21 +1066,21 @@ func assertDNSRecord(t *testing.T, recordName types.NamespacedName) error {
 
 	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 1*time.Minute, false, func(context context.Context) (bool, error) {
 		if err := kclient.Get(context, recordName, dnsRecord); err != nil {
-			t.Logf("Failed to get DNSRecord %v: %v; retrying...", recordName, err)
+			t.Logf("[%s] Failed to get DNSRecord %v: %v; retrying...", time.Now().Format(time.DateTime), recordName, err)
 			return false, nil
 		}
 		// Determine the current state of the DNSRecord.
 		if len(dnsRecord.Status.Zones) > 0 {
 			for _, zone := range dnsRecord.Status.Zones {
 				for _, condition := range zone.Conditions {
 					if condition.Type == v1.DNSRecordPublishedConditionType && condition.Status == string(metav1.ConditionTrue) {
-						t.Logf("Found DNSRecord %v %s=%s", recordName, condition.Type, condition.Status)
+						t.Logf("[%s] Found DNSRecord %v %s=%s", time.Now().Format(time.DateTime), recordName, condition.Type, condition.Status)
 						return true, nil
 					}
 				}
 			}
 		}
-		t.Logf("Found DNSRecord %v but could not determine its readiness; retrying...", recordName)
+		t.Logf("[%s] Found DNSRecord %v but could not determine its readiness; retrying...", time.Now().Format(time.DateTime), recordName)
```

Comment:
Can we have a clear log when `len(dnsRecord.Status.Zones) == 0` ?
For example on OpenStack, the `Status` is empty even it has `Spec`.
```
$ oc -n openshift-ingress get dnsrecords gateway-586fb57b47-wildcard -oyaml | yq .spec
dnsManagementPolicy: Managed
dnsName: '*.gw.d5p8jv2-b5564.shiftstack.devcluster.openshift.com.'
recordTTL: 30
recordType: A
targets:
  - 38.102.83.6

$ oc -n openshift-ingress get dnsrecords gateway-586fb57b47-wildcard -oyaml | yq .status
null
```

Reply:
Done.